drivers/hmis/app/graphql/schema.graphql

```diff
@@ -277,6 +277,11 @@ type Assessment {
   enrollment: Enrollment!
   event: Event
   exit: Exit
+
+  """
+  Client Location where the assessment was performed
+  """
+  geolocation: Geolocation
```

Comment:
@martha I liked your point about naming. Maybe these should all be `clientLocation[s]`?
`location` on it's own feels like it may get confusing down the line with other types of locations (project locations, tagged locations/areas/places, etc?)

Reply:
clientLocation (or maybe something like clientPin, or clientCoordiantes) feels right to me too.

Reply:
I like clientLocation too!

Reply:
sticking with geolocation per dev standup (geolocation terminology already used in form api; has searchability benefit; didn't seem like there was a very strong pref)